getting_started/scripting/gdscript/static_typing.rst

```diff
@@ -327,7 +327,7 @@ script editor’s status bar. The example below has 3 warnings:
    warning system example
 
 To ignore specific warnings in one file, insert a special comment of the
-form ``#warning-ignore:warning-id``, or click on the ignore link to the
+form ``# warning-ignore:warning-id``, or click on the ignore link to the
```

Comment:
I don't know if it's desirable to start warning ignore code with spaces. I prefer to use a space at the beginning of English comments, but not have a space for code, so that I can tell which is which. I would put ignoring warnings into that second category.

Reply:
I think this is a matter of taste. Personally I prefer the space to be there because it makes the text more readable. I never thought that anyone would like it withour the space better, so I just "fixed" it.

Should I remove the space here?